iconservice/base/type_converter.py

```diff
@@ -263,15 +263,23 @@ def convert_data_params(annotations: dict, kw_params: dict) -> None:
             kw_params[param_name] = kw_param
 
     @staticmethod
-    def adjust_params_to_method(func: callable, kw_params: dict):
+    def adjust_params_to_method(func: callable, kw_params: dict, remove_invalid_param: bool = False):
         hints = TypeConverter.make_annotations_from_method(func)
 
         # check user input argument name is valid
+        invalid_keys = []
         for key in kw_params.keys():
             try:
                 _type = hints[key]
             except KeyError:
-                raise InvalidParamsException(f"Invalid parameter name '{key}'")
+                invalid_keys.append(key)
```

Comment:
I think this logic needs comments about history. Without comments, It could be hard to understand why allows the invalid keys before Revision 9

Reply:
Thanks for your review

I made 2 modifications.

1. I extended functionality of adjust_params_to_method(). You can make choice its functionality with remove_invalid_params parameter.

2. With Revision and To address, I set proper remove_invalid_params for backward compatibility when call adjust_params_to_method().

Thus, I thought there is no needs to explain why remove_invalid_params was added and its history